benchmarks/src/generated/main/grpc/io/grpc/benchmarks/proto/BenchmarkServiceGrpc.java

```diff
@@ -157,6 +157,12 @@ protected BenchmarkServiceFutureStub build(io.grpc.Channel channel,
     }
   }
 
+  public static abstract class AbstractBenchmarkService implements BenchmarkService, io.grpc.stub.BindableService {
```

Comment:
I am having a hard time understanding what the value add is here.  Would you be able to add examples classes that show off how this would be used?  Right now this just seems to be a minor convenience class.

I am pretty reluctant to increase our API surface without some real, tangible benefit.